rules/S6425/javascript/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "title": "Import type should be used when possible",
```

```suggestion
  "title": "Type imports should be used when possible",
```

I would also rename the corresponding ticket in SonarJS.

---

...as well as the PR title...